### DISCUSSION TITLE
Soul Bounded Token (AEIP10)

### DISCUSSION THREAD

**Discussion by samuelmanzanera** [archethic-foundation/aeip#36] [AEIP]
This is a discussion for the AEIP10 located at: https://github.com/archethic-foundation/aeip/blob/main/AEIP-10.md

**Comment by redDwarf03** [comment]
1) This AEIP should be consistent with AEIP-6 regarding token archiving.
2) About revocation, specify a timestamp instead of a date (I guess it was an example... ).
3) About the revocation, it could be interesting to add a scenario with an oracle value. For example, if the value of oracle X is not included between A and B, then the SBT is revoked. The oracle could describe an exogenous event for example. So it could be interesting to revoke all SBT from a specific event

**Reply by samuelmanzanera** [reply]
About the AEIP 6 and the archiving , i think it should be agnostics of the token types: fungible, NFT, SBT, etc. 
I didn't get the consistency issue you mentioned ..

And about the revocation based on complex conditions such as oracle, in my opinion this is more related about the UTXO hooks capability. Also, the oracle could not invalid  tokens on itself, but the usage or the status's  check of the SBT can lead to a revocability.
